### DISCUSSION TITLE
Embed .net or JS  or Python  web tech inside  superset dashboard ?

### DISCUSSION THREAD

**Discussion by alnaimiab** [apache/superset#19303] [General]
how can I embed a web page  into a dashboard or any kind of programing logic ??

**Comment by srinify** [comment]
Hey there, Superset is a data viz & exploration tool and embedding web pages is traditionally out of scope for this class of tools.

You _can_ do limited HTML + CSS by using the Markdown component when editing a dashboard. This lets you do things like:

- Write markdown and have it render nice HTML + CSS
- Write raw HTML + CSS code to do things like fetch & load external images
